tests/suites/test_suite_aes.function

```diff
@@ -354,9 +354,12 @@ void aes_encrypt_ofb( int fragment_size, char *hex_key_string,
     TEST_ASSERT( strlen( hex_src_string ) <= ( 64 * 2 ) );
     TEST_ASSERT( strlen( hex_dst_string ) <= ( 64 * 2 ) );
 
-    key_len = mbedtls_test_unhexify( key_str, hex_key_string );
-    mbedtls_test_unhexify( iv_str, hex_iv_string );
-    in_buffer_len = mbedtls_test_unhexify( src_str, hex_src_string );
+    TEST_ASSERT( mbedtls_test_unhexify( key_str, sizeof( key_str ),
+                                        hex_key_string, &key_len ) == 0 );
```

Comment:
Not changed, it would be a bit shorter but I am not a fan of such complicated macros.

Reply:
The reason I favor a macro is that it would ensure that there's no mismatch between the array and its sizeof argument. (The macro should include a static assertion that the first argument is an array.) But I won't insist on it.